Gemfile

```diff
@@ -12,6 +12,7 @@ group :test do
   gem "omniauth-openid", "~> 1.0.1"
   gem "webrat", "0.7.3", require: false
   gem "mocha", "~> 1.1", require: false
+  gem 'test_after_commit', require: false
```

Comment:
@allenwq I think we could use a third option, that is:

3. On Rails 5 the after commits callbacks will run. Today the Rails solve the fixtures tests using the `uses_transaction`method. We can see how they use in the PR that supports the after commits: https://github.com/rails/rails/pull/18458/files#diff-f51660d918ba5921aaae392122026533L1310

I never used this before, but I think it worth the try. What do you think?